app/controllers/users.js

```diff
@@ -0,0 +1,9 @@
+const { signup } = require('../services/users');
+const { crypt } = require('../helpers');
+
+exports.users = (req, res, next) => {
+  req.body.password = crypt(req.body.password);
+  signup(req.body)
+    .then(sign => res.status(200).send(sign))
```

Comment:
use res.status (201) when you create a resource, change `.then(sign => res.status(200).send(sign))` to `.then(sign => res.status(201).send(sign))`